### PR TITLE
chore: remove circuits from repo

### PR DESCRIPTION
**Pull request by cedoor**
## Description

Circuits have been moved to the [zk-kit.circom](https://github.com/privacy-scaling-explorations/zk-kit.circom) and [zk-kit.noir](https://github.com/privacy-scaling-explorations/zk-kit.noir) repositories to better organize the code. Thus, they will be removed from the main zk-kit repo.

## Related Issue(s)

Re #282

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [x] I have run `yarn style` without getting any errors
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/workflows/pull-requests.yml [hunk 1]
```diff
@@ -33,50 +33,24 @@ jobs:
...
-                  toolchain: 0.26.0
-
             - name: Install dependencies
               run: yarn
 
             - name: Build libraries and contracts
               run: yarn build
 
             - name: Test ${{ matrix.type }}
-              # this condition is to skip contract tests when PR changes are unrelated
```
Comment (cedoor) [inline .github/workflows/pull-requests.yml:80]: Removed as contracts depends on local JS libs. If there's a change in the `@zk-kit/imt` lib, it'd be better to test contracts too.

**Comment by 0xjei** [comment]
hello @cedoor - will review this afternoon!

File: .github/workflows/pull-requests.yml (hunk 1 above)
Comment (0xjei) [reply .github/workflows/pull-requests.yml:80]: makes sense for now!

File: README.md [hunk 2]
```diff
@@ -49,18 +49,24 @@
     </h4>
 </div>
 
-| ZK-kit is a set of libraries (plugins, algorithms or utility functions) that can be reused in different projects and zero-knowledge protocols, making it easier for developers to access user-friendly, tested, and documented libraries for common tasks. |
-| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ZK-Kit is a set of libraries (algorithms or utility functions) that can be reused in different projects and zero-knowledge protocols, making it easier for developers to access user-friendly, tested, and documented code for common tasks. ZK-Kit provides a repository for each language. This repository only contains JavaScript and Solidity code. |
```
Comment (0xjei) [inline README.md:52]:
```suggestion
| ZK-Kit is a set of libraries (algorithms or utility functions) that can be reused in different projects and zero-knowledge protocols, making it easier for developers to access user-friendly, tested, and documented code for common tasks. ZK-Kit provides different repositories for each language - this one contains JavaScript and Solidity code only. |
```

**Review by 0xjei — CHANGES_REQUESTED** [review]
Great work :fire: just small nits:
- main readme description
- we should remove the `"@zk-kit/eddsa-proof#build"` from `turbo.json` under 
```js
"//#_test:libraries": {
  "dependsOn": [
     // ...
  ]
}
```
- run `yarn` once again to update the `yarn.lock`

**Pull request merged**